modules/drivers/presto-jdbc/src/metabase/driver/presto_jdbc.clj

```diff
@@ -189,6 +189,40 @@
 ;;; |                                                  Connectivity                                                  |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+;;; Kerberos related definitions
+(def ^:private ^:const kerb-props->url-param-names
+  {:kerberos-principal "KerberosPrincipal"
+   :kerberos-remote-service-name "KerberosRemoteServiceName"
+   :kerberos-use-canonical-hostname "KerberosUseCanonicalHostname"
+   :kerberos-credential-cache-path "KerberosCredentialCachePath"
+   :kerberos-keytab-path "KerberosKeytabPath"
+   :kerberos-service-principal-pattern "KerberosServicePrincipalPattern"
+   :kerberos-config-path "KerberosConfigPath"})
+
+(defn- details->kerberos-url-params [details]
+  (let [remove-blank-vals (fn [m] (into {} (remove (fn [[_ v]]
+                                                       (or (nil? v) (= "" v))) m)))
```

Comment:
seems like a poor version of `str/blank?` unless you intentionally want to keep strings that might be several whitespace characters. `(remove (comp str/blank? val) ...)` is perhaps a bit better.

Reply:
Thanks; updated this accordingly.